zowe-cli-id-federation-plugin/src/api/templates/tss.jcl

```diff
@@ -0,0 +1 @@
+TSS ADD({{mainframe_id}}) IDMAP(zwednmap) IDMAPDN('{{#escape}}{{distributed_id}}{{/escape}}') IDMAPRN('{{#escape}}{{registry}}{{/escape}}') IDLABEL('dn user mapping')
```

Comment:
- IDLABEL - this is an optional comment. I suppose we should remove it or make it part of CSV. Or it can be an additional input parameter
- IDMAP - I am not sure what exactly belongs there. But the documentation says it should be a unique identifier.

Reply:
As for the IDMAP - it should be unique identifier (same concept is in ACF2) because TSS does not allow multiple mappings unless you distinguish them with the IDMAP identifier. We could assume that nobody will have mappings under `zwednmap` identifier before this tool. Question is, if it should be optional parameter of the command. In ACF2 the identifier is used within the USERID with dot notation .... like WEIPE03.zwednmap

Reply:
after discussion IDMAP was changed to uppercase ZWEDNMAP and IDLABEL to user name so it matches the RACF variation